en/mkdocs.yml

```diff
@@ -201,13 +201,13 @@ nav:
           - 'Integrating WSO2 Identity Server with Liferay': learn/integrating-wso2-identity-server-with-liferay.md
           - 'Writing XACML2.0 Policies in WSO2 Identity Server':
             - 'Writing XACML2.0 Policies in WSO2 Identity Server': learn/writing-xacml2.0-policies-in-wso2-identity-server.md
-            - 'Writing XACML policies in WSO2 Identity Server - 1': learn/writing-xacml-policies-in-wso2-identity-server-1.md
-            - 'Writing XACML policies in WSO2 Identity Server - 2': learn/writing-xacml-policies-in-wso2-identity-server-2.md
-            - 'Writing XACML policies in WSO2 Identity Server - 3': learn/writing-xacml-policies-in-wso2-identity-server-3.md
-            - 'Writing XACML policies in WSO2 Identity Server - 4': learn/writing-xacml-policies-in-wso2-identity-server-4.md
-            - 'Writing XACML policies in WSO2 Identity Server - 5': learn/writing-xacml-policies-in-wso2-identity-server-5.md
-            - 'Writing XACML policies in WSO2 Identity Server - 6': learn/writing-xacml-policies-in-wso2-identity-server-6.md
-            - 'Writing XACML policies in WSO2 Identity Server - 7': learn/writing-xacml-policies-in-wso2-identity-server-7.md
+            - 'Introduction to XACML Policies': learn/introduction-to-xacml-policies.md
```

Comment:
We have checked if these files have been referred to in other places right?

Reply:
yes